neurobooth_os/tasks/task.py

```diff
@@ -361,7 +363,66 @@ def doDriftCorrect(self, vals):
         if self.eye_tracker is not None:
             self.eye_tracker.tk.doDriftCorrect(*vals)
 
-    def update_tablet_background(self, x: float, y: float) -> None:
+    def gaze_contingency():
+        # move task
+        pass
+
+
+class Color(Enum):
+    BLACK = 0
+    RED = 12
+    GREEN = 10
+
+
+class Eyelink_HostPC(Task_Eyetracker):
```

Comment:
I'm not sure this class should be a subtype of Task_Eyetracker, since it's not actually a "task".  It may make sense to have Task_Eyetracker have an instance of Host_PC, that provides this functionality.

Reply:
That's a good point, can we discuss this in the thread below with Brandon?

Reply:
Agree with points in Slack. Would prefer composition over inheritance in this case, but understand if we are avoiding this refactor for now.

Reply:
Thanks!